docs/reference/generated/field-validity.json

```diff
@@ -3,7 +3,7 @@
   "description": "Used to display a custom message based on the field’s validity.\nRequires `children` to be a function that accepts field validity state as an argument.",
   "props": {
     "children": {
-      "type": "(validity) => React.ReactNode",
+      "type": "((state: FieldValidityState) => ReactNode)",
```

Comment:
In a follow up we may want to improve and show the values in this enums, I saw similar one around the `ImageLoadingStatus`